mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactTableView.js

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { getSrc } from './ShowArtifactPage';
+import { getArtifactContent } from '../../../common/utils/ArtifactUtils';
+import { Table } from '@databricks/design-system';
+import { FormattedMessage } from 'react-intl';
+import Papa from 'papaparse';
+
+const ShowArtifactTableView = ({ runUuid, path, getArtifact }) => {
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState();
+  const [data, setData] = useState();
+  const [headers, setHeaders] = useState();
+  const [text, setText] = useState();
+
+  const MAX_ROW_LENGTH = 500;
```

Comment:
```suggestion
  const MAX_NUM_ROWS = 500;
```

`MAX_ROW_LENGTH` sounds like the maximum number of elements in a row. What do you think?